intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -1132,49 +1135,18 @@ \subsection{Activating and de-activating modules}
 Modules need not be loaded one by one; the two ``load'' commands can be
 combined as follows:
 
-\ifantwerpen
-\begin{prompt}
-%\shellcmd{module load MATLAB Python/2.7.9-intel-2015a}%
-\end{prompt}
-\fi
-\ifbrussel
+\ifleuven
 \begin{prompt}
-%\shellcmd{module load MATLAB Python/2.7.9-intel-2015a}%
+%\shellcmd{module load matlab Python/2.7.6-foss-2014a}%
 \end{prompt}
-\fi
-\ifgent
+\else
 \begin{prompt}
 %\shellcmd{module load MATLAB Python/2.7.9-intel-2015a}%
```

Comment:
This is actually a bad example, people should always load modules with a specific version (and we should mention that somewhere in the documentation if it isn't yet).

Without specifying a version they will suddenly start loading a different (newer) version when we install a new module (by default it picks the latest available version if you don't specify one). That may lead to surprises because the job script may break all of a sudden (if the new software version is backwards-incompatible with previous version), and even if it doesn't it should be avoided since it conflicts with the idea of reproducible results.

Reply:
It is mentioned somewhere else in the docs, I forgot.